src/pages/public_registration/ActionPanel.tsx

```diff
@@ -19,23 +20,61 @@ export const ActionPanel = ({
   isLoadingData,
 }: ActionPanelProps) => {
   const { t } = useTranslation();
+  const [tabValue, setTabValue] = useState(0);
 
   return (
-    <Paper
-      elevation={0}
-      data-testid={dataTestId.registrationLandingPage.tasksPanel.panelRoot}
-      sx={{ gridArea: 'tasks' }}>
-      <StyledPaperHeader>
-        <Typography color="inherit" variant="h1">
-          {t('common.tasks')}
-        </Typography>
-      </StyledPaperHeader>
-      <ActionPanelContent
-        tickets={tickets}
-        refetchData={refetchRegistrationAndTickets}
-        isLoadingData={isLoadingData}
-        registration={registration}
-      />
+    <Paper elevation={0} data-testid={dataTestId.registrationLandingPage.tasksPanel.panelRoot}>
+      <Box sx={{ color: 'primary.contrastText' }}>
+        <Tabs
+          value={tabValue}
+          onChange={(_, newValue) => setTabValue(newValue)}
+          aria-label="action-panel-tabs"
+          sx={{ bgcolor: 'primary.main', px: '0.5rem' }}
+          textColor="inherit"
+          TabIndicatorProps={{ style: { backgroundColor: 'white', height: '0.4rem' } }}>
+          <Tab
+            label={t('common.tasks')}
+            data-testid={dataTestId.registrationLandingPage.tasksPanel.tabPanelTasks}
+            id="action-panel-tab-0"
+            aria-controls="action-panel-tab-panel-0"
+          />
```

Comment:
Tror ikke en skjermleser plukker opp `label` her, så kanskje legg til `aria-label`?
```suggestion
          <Tab
            label={t('common.tasks')}
            data-testid={dataTestId.registrationLandingPage.tasksPanel.tabPanelTasks}
            id="action-panel-tab-0"
            aria-controls="action-panel-tab-panel-0"
            aria-label="Some description"
          />
```